src/App.js

```diff
@@ -1,54 +1,43 @@
-/**
- * Copyright 2018 Google Inc. All Rights Reserved.
+/*
+ * Copyright 2019 Google LLC
+ *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
- *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
  * Unless required by applicable law or agreed to in writing, software
  * distributed under the License is distributed on an "AS IS" BASIS,
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
 
-import React, { Component, lazy, Suspense } from 'react';
+import React, { useEffect, lazy, Suspense, Fragment } from 'react';
 import { Route, withRouter } from 'react-router-dom';
 import quicklink from 'quicklink/dist/quicklink.mjs';
 
 import './App.css';
 
-const LazyLanding = lazy(() => import(/* webpackChunkName: "landing" */ './containers/Landing'));
-const LazyItemView = lazy(() => import(/* webpackChunkName: "item-view" */ './containers/ItemView'));
+const LazyLanding = lazy(() => import('./containers/Landing/Landing'));
+const LazySaleProducts = lazy(() => import('./containers/SaleProducts/SaleProducts'));
```

Comment:
I see we dropped the `webpackChunkName` entries from here. Was this intentional?

Reply:
It was a mistake, now fixed it.